cmd/storj-sim/network.go

```diff
@@ -281,6 +288,10 @@ func newNetwork(flags *Flags) (*Processes, error) {
 
 				"--version.server-address", fmt.Sprintf("http://%s/", versioncontrol.Address),
 				"--debug.addr", net.JoinHostPort("127.0.0.1", port(satellitePeer, i, debugHTTP)),
+
+				"--database", database,
+				"--schema", schema,
+				// TODO(jg): add connection to postgres for metainfo.databaseURL once metainfo refactor is complete
```

Comment:
You can add the pointerdb/metainfo url already now, there shouldn't be issues with it. If there are then these should be fixed.

Reply:
yeah i was having trouble with metainfodb using postgres, but let me try again. I could have been doing something wrong.